addons/base/views.py

```diff
@@ -313,14 +313,20 @@ def create_waterbutler_log(payload, **kwargs):
     with transaction.atomic():
         try:
             auth = payload['auth']
+            user = OSFUser.load(auth['id'])
             # Don't log download actions
             if payload['action'] in ('download_file', 'download_zip'):
+                if payload['action_meta']['is_mfr_render']:
+                    node = AbstractNode.load(payload['metadata']['nid'])
+                    if not node.is_contributor(user):
+                        file = BaseFileNode.load(payload['metadata']['path'].lstrip('/'))
```

Comment:
np: file is a builtin name in python, so not a good variable name choice, but shouldn't actually be hurting anything